### PR TITLE
Fixing log statement in EFolderService

### PR DESCRIPTION
**Pull request by mogrenAtWork**
- Connects #2340 
- Connects #2808

- Fixes string injection in MetricsService name to log the metrics for the eFolder request correctly
- Sets the certificate and SSL version in the eFolder request

### CODE REVIEW

File: app/services/external_api/efolder_service.rb [hunk 1]
```diff
@@ -33,7 +33,7 @@ def self.get_efolder_response(endpoint, user, headers = {})
     headers["STATION-ID"] = user.station_id.to_s
     request.headers = headers
 
-    MetricsService.record("eFolder GET request to ${url}",
+    MetricsService.record("eFolder GET request to #{url}",
```
Comment (shanear) [inline app/services/external_api/efolder_service.rb]: awkward 🐸

Comment (askldjd) [inline app/services/external_api/efolder_service.rb]: That's how I would have done it
MetricsService.record("#{console.log('$url')}")

File: app/services/external_api/efolder_service.rb [hunk 2]
```diff
@@ -27,13 +27,15 @@ def self.efolder_key
   def self.get_efolder_response(endpoint, user, headers = {})
     url = URI.escape(efolder_base_url + endpoint)
     request = HTTPI::Request.new(url)
+    request.auth.ssl.ssl_version  = :TLSv1_2
+    request.auth.ssl.ca_cert_file = ENV['SSL_CERT_FILE']
```
Comment (shanear) [inline app/services/external_api/efolder_service.rb]: did you test this?

Comment (kierachell) [inline app/services/external_api/efolder_service.rb]: by hotfixing the code in UAT :hurtrealbad:

**Pull request merged**